lms/djangoapps/courseware/model_data.py

```diff
@@ -24,6 +26,20 @@
 from xmodule.modulestore.django import modulestore
 from xblock.core import XBlockAside
 
+try:
+    DJANGO_FIELD_CACHE = get_cache('xblock-fields')
+except InvalidCacheBackendError:
+    DJANGO_FIELD_CACHE = get_cache('default')
+
+DJANGO_CACHED_FIELDS = [
+    ('sequential', 'position'),
+    ('chapter', 'position'),
+    ('problemset', 'position'),
+    ('videosequence', 'position'),
+    ('video', 'speed'),
+    ('video', 'saved_video_position'),
+]
+
```

Comment:
A commented explanation of wth `DJANGO_FIELD_CACHE` is and why these fields are in `DJANGO_CACHED_FIELDS` is a must here.